spec/views/comments/index.html.erb_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+RSpec.describe "comments/index", type: :view do
+  before(:each) do
+    assign(:comments, [
+      Comment.create!(
+        :body => "Body"
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)